cli/cli.cpp

```diff
@@ -0,0 +1,150 @@
+#include <iostream>
+#include <string>
+#include <exception>
+#include <stdexcept>
+#include "display.h"
+
+void help() {
+    std::cout << "Rotate and rearrange displays\n";
```

Comment:
instead of managing the command line input yourself you coul have a look at https://github.com/daniele77/cli
Which might be a bit overkill for this usecase tho.

Reply:
lovely, will be usefull in the future. for this it is overkill tho